lib/commands/serve.js

```diff
@@ -21,7 +21,7 @@ function serveCommand(argv, context) {
     return;
   }
 
-  if (context.type !== 'app' && context.type !== 'platform') {
+  if (context.type !== 'app' && context.type !== 'platform' && context.type !== 'settings') {
```

Comment:
Maybe use the new `context.isUiModule` property.  It is intended to consolidate testing for the various `ui-*` types like app/settings/plugin, etc.  Also makes some progress on STCLI-78

Reply:
🤔 do we want to be able to `serve`, say `stripes-core`, or should I continue to throw that warning when the `context.isStripesModule`?

Reply:
Oh I didn't see this until after...  Probably should still throw on isStripesModule

Reply:
We are going to need to allow `type === platform` as well.